src/index.html

```diff
@@ -13,10 +13,14 @@
     <title>Stop watch</title>
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="./main/main.scss"
```

Comment:
Fix folder name with styles
```suggestion
      href="./styles/main.scss"
```